base/operators.jl

```diff
@@ -1149,40 +1149,68 @@ julia> filter(!isletter, str)
 !(f::ComposedFunction{typeof(!)}) = f.inner #allows !!f === f
 
 """
-    Fix1(f, x)
+    Fix{n}(f, x)
+    Fix{kw}(f, x)
+    Fix(f; [kw=]x)
```

Comment:
```suggestion
    Fix(f; kw=x)
```
See https://github.com/JuliaLang/julia/pull/54653#discussion_r1666031846